.github/workflows/workflow.yml

```diff
@@ -3,6 +3,7 @@ name: CI
 env:
   RUST_VERSION: 1.76.0
   DFX_VERSION: 0.21.0
+  POCKET_IC_VERSION: 7.0.0
```

Comment:
NOTE: for some reason having 6.0.0 was not enough and the test was failing with

```shell
Installing the bitcoin canister...
thread 'canbench_is_not_in_bitcoin_canister' panicked at /home/maksym/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pocket-ic-6.0.0/src/lib.rs:1518:9:
Incompatible PocketIC server version: got pocket-ic-server 6.0.0; expected pocket-ic-server 7.0.0.
stack backtrace:```

so I used 7.0.0 here.

Reply:
Yes, the server's version is not the library's version. At the moment they are off by one, which makes it a bit more confusing than if they were off by more.

Reply:
So I think it would be good to call this constant POCKET_IC_SERVER_VERSION

Reply:
Also, canbench already downloads the correct version for you. Does this repo have its own suite of pocketic tests?

Reply:
> I think it would be good to call this constant POCKET_IC_SERVER_VERSION

done.